st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -69,9 +69,6 @@ def get_one(self, name, requester_user, scope=FULL_SYSTEM_SCOPE, user=None, decr
             Handle:
                 GET /keys/key1
         """
-        if not scope:
-            scope = FULL_SYSTEM_SCOPE
-
```

Comment:
Are you sure about this change?

Reply:
Left over from when I was experimenting w/ st2api. Removed.